### PR TITLE
RCB-627: create rosette-api-all to combine model and json modules

### PR DESCRIPTION
**Pull request by Katsuya-Tomioka**

### CODE REVIEW

File: all/pom.xml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    <parent>
+        <groupId>com.basistech.rosette</groupId>
+         <artifactId>rosette-api-java-binding</artifactId>
+        <version>1.31.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>rosette-api-all</artifactId>
+    <name>rosette-api-all</name>
+    <description>Rosette API Communications</description>
+    <properties>
+        <skip-mockserver>false</skip-mockserver>
```
Comment (asoosprecognox) [inline all/pom.xml]: Is this property needed for this module?

File: all/pom.xml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.basistech.rosette</groupId>
+         <artifactId>rosette-api-java-binding</artifactId>
+        <version>1.31.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>rosette-api-all</artifactId>
+    <name>rosette-api-all</name>
+    <description>Rosette API Communications</description>
```
Comment (asoosprecognox) [inline all/pom.xml]: Could this be changed to something specific, or at least not the same thing as the api module?

**Review by asoosprecognox — COMMENTED** [review]
Works great!
Is the fact that the new module is not added to the root pom intentional?

**Comment by Katsuya-Tomioka** [comment]
Updated pom, and jenkins is happy.

**Review by asoosprecognox — APPROVED** [review]

**Pull request merged**